emcc.py

```diff
@@ -1139,6 +1139,9 @@ def check(input_file):
       options.js_libraries.append(shared.path_from_root('src', 'library_fetch.js'))
       if shared.Settings.USE_PTHREADS:
         shared.Settings.FETCH_WORKER_FILE = unsuffixed(os.path.basename(target)) + '.fetch.js'
+        # This is a hack for make_fetch_worker(), which requires
+        # emscripten_is_main_runtime_thread, which is called from emscripten_fetch()
```

Comment:
not sure i understand the comment - is this keeping one function alive in order to retain a function that it calls? why not just export `emscripten_is_main_runtime_thread` if that's all we need?

Reply:
It seemed like keeping `emscripten_fetch()` was aligned with the expectations of `make_fetch_worker()`. I'll change it.